UPDATING.md

```diff
@@ -24,10 +24,73 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+## Changes in Google Cloud Platform related operators
+
+Most GCP-related operators have now optional `PROJECT_ID` parameter. In case you do not specify it,
+the project id configured in
+[GCP Connection](https://airflow.apache.org/howto/manage-connections.html#connection-type-gcp) is used.
+There will be an `AifrlowException` thrown in case `PROJECT_ID` parameter is not specified and the
+connection used has no project id defined. This change should be  backwards compatible as earlier version
+of the operators had `PROJECT_ID` mandatory.
+
+Operators involved:
+
+  * GCP Bigtable Operators
+    * BigtableInstanceCreateOperator
+    * BigtableInstanceDeleteOperator
+    * BigtableTableCreateOperator
+    * BigtableTableDeleteOperator
+    * BigtableClusterUpdateOperator
+    * BigtableTableWaitForReplicationSensor
+  * GCP Compute Operators
+    * GceInstanceStartOperator
+    * GceInstanceStopOperator
+    * GceSetMachineTypeOperator
+    * GceInstanceTemplateCopyOperator
+    * GceInstanceGroupManagerUpdateTemplateOperator
+  * GCP Function Operators
+    * GcfFunctionDeployOperator
+  * GCP Spanner Operators
+    * CloudSpannerInstanceDeployOperator
+    * CloudSpannerInstanceDeleteOperator
+    * CloudSpannerInstanceDatabaseQueryOperator
+    * CloudSpannerInstanceDatabaseDeployOperator
+    * CloudSpannerInstanceDatabaseUpdateOperator
+    * CloudSpannerInstanceDatabaseDeleteOperator
+  * GCP Cloud SQL Operators
+    * CloudSqlInstanceCreateOperator
+    * CloudSqlInstancePatchOperator
+    * CloudSqlInstanceDeleteOperator
+    * CloudSqlInstanceDatabaseCreateOperator
+    * CloudSqlInstanceDatabasePatchOperator
+    * CloudSqlInstanceDatabaseDeleteOperator
+    * CloudSqlInstanceExportOperator
+    * CloudSqlInstanceImportOperator
+    * CloudSqlQueryOperator (project_id might be omitted from the extras of cloudsql URL)
```

Comment:
To be honest we don't require this as these operators were added to master and were not part of 1.10.1.

Reply:
I think @potiuk added under master section for the file. Hence it implicitly implies these operators / change to the operators are only available in master branch.

Reply:
@feng-tao  I think what @kaxil refers to is that the master part of the UPDATING.md file will go to 1.10.2 part of the release so technically that's correct - those operators were not released yet in 1.10.1 so there is nothing to update for them :). They are just new operators for the users. I will remove those references then.